### PR TITLE
Contributing to App Store 1.1

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

Update to the new app store structure

## Type of change

- [x] This change requires a documentation update

## How should this be tested?

- Read through new changes

### CODE REVIEW

File: apps/docs/tsconfig.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "extends": "@calcom/tsconfig/nextjs.json",
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "noImplicitAny": false
+  }
```
Comment (joeauyeung) [inline apps/docs/tsconfig.json:7]: Only changed here for docs since most files and mdx

**Comment by PeerRich** [comment]
huh @joeauyeung  this is empty now
image

**Review by zomars — CHANGES_REQUESTED** [review]
All changes disappeared. Let's try to fix that or open a new PR if necessary. Then you can re-request a review @joeauyeung

**Comment by joeauyeung** [comment]
@zomars interesting. I think when Peer merged to main it removed the number of files changed but I'm seeing the changes here: https://docs.cal.com/developer/app-store. If you see any changes that need to be made I'll create a new PR.

**Pull request closed without merging**